pet-server/models/usersModel.js

```diff
@@ -0,0 +1,48 @@
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../database/databaseConnection");
+
+const User = sequelize.define(
+  "user",
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    username: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [3, 20],
+      },
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true,
+      validate: {
+        isEmail: true,
+      },
+    },
+    mobileNumber: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true,
+      validate: {
+        len: [10, 15],
+      },
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [8, 72],
+      },
+    },
+  },
+  {
+    tableName: "users",
+  }
```

Comment:
sequelize,
      tableName: 'user',
      underscored: true

Reply:
enable underscored true and added single quote. push the changes